packages/react-storage/src/components/StorageBrowser/composables/Search.tsx

```diff
@@ -68,23 +65,11 @@ export const Search = ({
       </Field>
       <ButtonElement
         className={`${BLOCK_NAME}__submit-button`}
-        onClick={() => onSearch?.(term, subfoldersIncluded)}
+        onClick={() => onSearch?.()}
       >
         Submit
       </ButtonElement>
-      {showIncludeSubfolders ? (
-        <SpanElement className={`${BLOCK_NAME}-${TOGGLE_BLOCK}__container`}>
-          <LabelElement className={`${BLOCK_NAME}-${TOGGLE_BLOCK}__label`}>
-            <InputElement
-              checked={subfoldersIncluded}
-              className={`${BLOCK_NAME}-${TOGGLE_BLOCK}__checkbox`}
-              onChange={() => setSubfoldersIncluded(!subfoldersIncluded)}
-              type="checkbox"
-            />
-            Include subfolders
-          </LabelElement>
-        </SpanElement>
-      ) : null}
+      {children ?? null}
```

Comment:
Since we have decoupled the subfolder checkbox we should remove `children` and let the parent view wrap the controls for positioning

Reply:
I know we're trying to get rid of the wrapping divs, I can remove those and the `children` from Search at the same time in a follow up, if that makes sense.